ui/main/src/assets/i18n/en.json

```
@@ -113,7 +113,8 @@
     "at": "at",
     "from" : "from",
     "moreCards": "cards more",
-    "lastResponse": "Last response received on"
+    "lastResponse": "Last response received on",
+    "cardLoadingInProgress":"Loading card in progress ..."
```

```suggestion
    "cardLoadingInProgress":"Loading card ..."
```

I think the `ing` already conveys that the action is currently being done. Alternatively, if we want to use `in progress` then what precedes has to be some kind of noun, so `Card loading in progress`.

---

ok done